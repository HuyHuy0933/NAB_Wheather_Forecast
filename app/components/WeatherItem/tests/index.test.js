import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { WeatherItem } from '../index';

const renderer = new ShallowRenderer();

describe('<WeatherItem />', () => {
  it('should render and match the snapshot', () => {
    const intl = {
      formatMessage: jest.fn(),
    };

    const weather = {
      applicableDate: '05-23-2020',
      minTemp: 28,
      maxTemp: 30,
    };
    renderer.render(<WeatherItem intl={intl} weather={weather} />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
