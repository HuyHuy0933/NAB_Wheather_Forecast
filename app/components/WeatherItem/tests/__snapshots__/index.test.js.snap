// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherItem /> should render and match the snapshot 1`] = `
<ForwardRef>
  <ForwardRef>
    Saturday
  </ForwardRef>
  <ForwardRef>
    <ForwardRef />
    28
  </ForwardRef>
  <ForwardRef>
    <ForwardRef />
    30
  </ForwardRef>
</ForwardRef>
`;
